pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceSummaryDto.java

```diff
@@ -108,4 +111,20 @@ public boolean isHidden() {
     public InstanceStatusType getStatusType() {
         return statusType;
     }
+
+    public int getInstanceNumber() {
+        return instanceNumber;
+    }
+
+    public void setInstanceNumber(int instanceNumber) {
+        this.instanceNumber = instanceNumber;
+    }
+
+    public String getPreviousInstanceGuid() {
+        return previousInstanceGuid;
```

Comment:
What is the purpose of `previousInstanceGuid`?

Reply:
I believe this was added for some features in A-T study.

Reply:
![3478d5e76ea255330a6d2ba27814cf19](https://user-images.githubusercontent.com/29984380/109033092-48969e80-7694-11eb-961d-3bef98f972e8.jpeg)

Reply:
Just to be clear, this property already existed before this PR. The changes here is populating this property so we don't break existing functionality.